### PR TITLE
💚 only deploy when lastUpdate is not up to date

### PR DESCRIPTION
**Pull request by thefrontendwizard**

### CODE REVIEW

File: .github/workflows/lastUpdate.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "lastUpdate": "hopefully-this-will-be-updated-when-action-runs" }
```
Comment (WyriHaximus) [inline .github/workflows/lastUpdate.json:1]: To be honest, I would put this file in `/etc/` of your repo root, or maybe `/utils/`? It's now inside a directory that isn't directly it's purpose

File: .github/workflows/deployment.yml [hunk 2]
```diff
@@ -16,12 +16,23 @@ jobs:
...
+      - name: Update timestamp
+        run: jq --arg timestamp steps.release.datasource.timestamp '.lastUpdate = $timestamp' .github/workflows/lastUpdate.json
+        if: steps.release.datasource.timestamp != steps.release.deployed.timestamp
+      - name: Commit changes
+        uses: EndBug/add-and-commit@v4
+        with:
+          message: ":green_heart: update timestamp"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        if: steps.release.datasource.timestamp != steps.release.deployed.timestamp
```
Comment (WyriHaximus) [inline .github/workflows/deployment.yml:38]: Good, this was the `Ow wait`, `Nvm` in our chat. Doing another commit would trigger this again, but Github Actions won't trigger other Github Actions unless you use a personal token.

File: .github/workflows/lastUpdate.json (hunk 1 above)
Comment (thefrontendwizard) [reply .github/workflows/lastUpdate.json:1]: The main reason I left in there is cause it feels it's a workflow only related file and I rather have them together than have in a place where it's not transparent that this is the purpose of it.

Comment (WyriHaximus) [reply .github/workflows/lastUpdate.json:1]: Fair point, it just feels weird to me 🤣.

**Pull request merged**